### DISCUSSION TITLE
Add review comments like SwaggerHub

### DISCUSSION THREAD

**Discussion by bperiasamy** [ApicurioArchive/apicurio-studio#2043] [Ideas]
If we have the option to add reviews/comments by the line number like the swagger Hub, it';ll be useful for team collaboration.

**Comment by EricWittmann** [comment]
Agreed.  That's on the wishlist!  I can't commit yet.  We'll see how things evolve.  :(

**Comment by bperiasamy** [comment]
Sure, Thanks Eric.
